### PR TITLE
Task 4 updates to Sessions & Users controllers, polish Requests index view

### PR DESCRIPTION
**Pull request by fkhayes**
This branch incorporates changes I made based on Xtol site mentor feedback.

By the way, based on Hartl's Rails Tutorial, my controllers use helpers related to logging-in, authentication, etc. that are defined in a sessions_helper module in the helpers directory. Hartl advises requiring the sessions_helper in application_controller.rb

### CODE REVIEW

File: app/views/requests/index.html.erb [hunk 1]
```diff
@@ -1,21 +1,23 @@
 <h1>Listing requests</h1>
 <div>
   <% @requests.each do |request| %>
-    <p><%= request.title %>: <%= request.start_date %> - <%= request.end_date %></p>
-    <p>Posted by: <a href="mailto:<%= request.email %>"><%= request.contact %></a> 
-    in <%= request.office.location %> (Group: <%= request.group.name %>)</p>
-    <p>Description: <%= request.description %></p>
-    <p>Relevant Skills: 
+    <div><%= request.title %>: <%= request.start_date.to_s(:long) %> -
```
Comment (v-ramachandran) [inline app/views/requests/index.html.erb]: In my original review, I didn't mean change all of these to divs. Change only the ones that group to divs. If its simply a block of text p is just fine.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -58,10 +46,7 @@ def update
   # DELETE /users/1.json
   def destroy
     @user.destroy
```
Comment (v-ramachandran) [inline app/controllers/users_controller.rb]: What happens if destroy fails?

**Comment by v-ramachandran** [comment]
I've noticed that you use the SessionHelper in the ApplicationController. Generally, Helpers are included into views automatically. If you have controller only items, why not just declare specific modules under lib?

**Comment by fkhayes** [comment]
Re: SessionHelper,
I've read that helpers are intended to bridge the gap between controllers and views, so I had wondered if there was a customary name and place for storing shared controller methods such as those found in SessionHelper for log-in, etc. Your suggestion to use the lib directory makes sense.

For now, I'd like to leave the functions in the SessionHelper in case it will make it easier for me to reference Hartl's tutorial in the future. I guess he references them in his views for some reason, maybe in the long run I won't.

**Pull request merged**